core/src/main/java/io/ably/lib/transport/ConnectionManager.java

```diff
@@ -1447,6 +1429,27 @@ public QueuedMessage(ProtocolMessage msg, CompletionListener listener) {
         }
     }
 
+    /**
+     * Send all pending messages which are queue.
+     * Remove them from the queue once they are send successfully
```

Comment:
A small improvement to the docs
```suggestion
     * Send all pending messages which are in the queue.
     * Remove them from the queue once they are sent successfully
```